server/routerlicious/packages/routerlicious-base/src/alfred/routes/api/documents.ts

```diff
@@ -227,6 +235,8 @@ export function create(
 						isSessionAlive: false,
 						isSessionActive: false,
 					};
+					// if undefined and added directly to the session object - will be serialized as null in mongo which is undesirable
```

Comment:
nit: where is the session object serialized in Mongo?

Reply:
In this case it's not that important and it can be inlined. But I decided to assign it outside in case someone's refactor this code and decides to add db write of this object. So just to be on a safe side :)